### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by SUPERCILEX**
See: https://github.com/travis-ci/travis-ci/issues/5582

### CODE REVIEW

**Comment by SUPERCILEX** [comment]
I wonder why trusty doesn't work.

**Comment by SUPERCILEX** [comment]
Apparently trusty doesn't have android tools.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+# See https://github.com/travis-ci/travis-ci/issues/5582
+sudo: required
+# See https://docs.travis-ci.com/user/languages/java/#Caching
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
```
Comment (SUPERCILEX) [inline .travis.yml]: I added caching while I was at it.

**Comment by samtstern** [comment]
LGTM, squashing and merging.

**Pull request merged**